### ISSUE TITLE
False positives with negating parantheses on number_separator rule

### ISSUE THREAD

**Issue by Dschee** [realm/SwiftLint#2683]
For this code
```swift
let x: Double = -(200 / 447.214)
```
I'm getting a `number_separator` rule violation which I don't expect. 

I'm using the default configuration for `number_separator` with SwiftLint at version 0.31.0.

**Comment by daltonclaybrook** [comment]
Looking into this.
